web/src/components/inventory/ShowNearby.tsx

```diff
@@ -0,0 +1,67 @@
+import { useState } from 'react';
+import useNuiEvent from '../../hooks/useNuiEvent';
+import { giveTo } from '../../dnd/onGive';
+import { useExitListener } from '../../hooks/useExitListener';
+
+interface Player {
+  player: number;
+}
+
+const ShowNearby: React.FC<{ isInventoryOpen: boolean }> = ({ isInventoryOpen }) => {
+  const [nearbyVisible, setNearbyVisible] = useState(false);
+  const [nearbyPlayers, setNearbyPlayers] = useState([]);
+  const [currentSlot, setSlot] = useState(0);
+  const [currentAmount, setAmount] = useState(0);
```

Comment:
These can probably be a state object since you are setting them all at once, no need for 3 separate states.

Reply:
nearbyVisible is in the block but that's fine as is, it can be it's own state.

Reply:
Done made changed it as an object in the latest commit